server/server.py

```diff
@@ -24,34 +24,37 @@ def handle(self):
 		try:
 			msg = self.RSAcipher.decrypt(data).decode().split(" ", 1)
 
-			if (msg[0] == '3317BLT5'):
+			if (msg[0] != '3317BLT5'):
+				raise BaseException("Invalid magic number")
+
 			with open("/var/log/locationLog", "a") as logFile:
 				date = time.localtime()
 				logFile.write("{0}  {1:02} {2:02}:{3:02}:{4:02}  {6} Checking in at {5}\n".format(months[date.tm_mon - 1], date.tm_mday, date.tm_hour, date.tm_min, date.tm_sec, self.client_address[0], msg[1].strip()))
 				print("[+] IP Logged to file")
+			
+		except BaseException as e:
+			print("[-] Failure: {}".format(e))
```

Comment:
Do these print statements go anywhere other than stdout? If this is running in the background of some server, it would be better for these to also go to a log file.